src/bin/firebase.ts

```diff
@@ -161,7 +164,7 @@ if (!handlePreviewToggles(args)) {
 
   // determine if there are any non-option arguments. if not, display help
   args = args.filter((arg) => {
-    return arg.indexOf("-") < 0;
+    return !arg.includes("-");
   });
   if (!args.length) {
```

Comment:
Oh man, I hate when I'm like "but you can make it _even better_". totally optional, and I'm 90% confident this is right (please check), but:

```typescript
const hasNotFlagParams = args.some((a) => a.includes("-"));
if (!hasNotFlagParams) {
```

Reply:
I think I hit the 10% in finding they are not functional equivalent. 

```
// before
args = ['x', '-']
args = args.filter((arg) => !arg.includes("-")); // ['x']
!x.length; // false

// after
args = ['x', '-']
hasNotFlagParams =args.some((a) => a.includes("-"));
!hasNotFlagParams; // true
```

Reply:
I think I'm going to leave as-is for now. (The code change happened automagically from IntelliJ 🤷 )